src/main/java/com/gluonhq/substrate/Constants.java

```diff
@@ -138,6 +139,7 @@
     public static final String DALVIK_PRECOMPILED_CLASSES = "/precompiled/classes/";
     public static final String DALVIK_ACTIVITY_PACKAGE = "com/gluonhq/helloandroid/";
     public static final String DALVIK_JAVAFX_PACKAGE = "javafx/scene/input/";
+    public static final String DALVIK_PROJECT_PATH = "/native/android/android_project";
```

Comment:
Can you refactor the path `/native/android/` to one constant, and then only the name to another one? There are few uses for the former and also for the latter.

Reply:
Fixed